pvr.waipu/resources/language/resource.language.de_de/strings.po

```diff
@@ -10,6 +10,14 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
+msgctxt "Addon Summary"
+msgid "waipu.tv PVR Client"
+msgstr "waipu.tv PVR Client"
+
+msgctxt "Addon Description"
+msgid "waipu.tv PVR Client. Note: This is not an official waipu plugin and not related to Exaring AG or waipu.tv. It was developed by volunteers."
+msgstr "waipu.tv PVR Client. Hinweis: Dies ist kein offizielles Waipu-Plugin und steht nicht in Verbindung zur Exaring AG oder zu waipu.tv. Es wurde von Freiwilligen entwickelt."
+
```

Comment:
Where is this used? Or why did you copy it from `addon.xml.in`?

Reply:
Needed to have available to https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-waipu/ for translation, the https://github.com/flubshi/pvr.waipu/blob/Nexus/.github/workflows/sync-addon-metadata-translations.yml bring them then back to addon.xml.in after change.

Reply:
Seen this https://github.com/flubshi/pvr.waipu/blob/Nexus/.github/workflows/sync-addon-metadata-translations.yml#L7 must also be fixed, then addon.xml on line should be addon.xml.in.